src/Avalonia.Layout/LayoutHelper.cs

```diff
@@ -9,6 +9,12 @@ namespace Avalonia.Layout
     /// </summary>
     public static class LayoutHelper
     {
+        /// <summary>
+        /// Epsilon value used for certain layout calculations.
+        /// Based on the value in WPF LayoutDoubleUtil.
+        /// </summary>
+        public const double LayoutEpsilon = 0.00000153;
```

Comment:
Hello, sorry for the delay, I just didn’t even understand earlier why I received a notification about this PR :)

Unfortunately, I have nothing to add to the comments from the wpf, but thanks for the info

I would only like to suggest using an `auto-property` here (we can also repalace to property `DoubleEpsilon` in doubleutils). Using a `const` in a public doesn't look good

Reply:
Thanks, no worries - I was simply interested if you have any insights on this. It seems that WPF number is just picked empirically and seems to work fine.

I've changed `LayoutEpsilon` to a property as well, makes sense from the ABI perspective. Values in our other utils are private/internal so we can keep it as const for now.